tests/integration/test_command_line_client.py

```diff
@@ -571,6 +577,39 @@ def test_command_get_recursive_and_query():
         assert filecmp.cmp(downloaded, uploaded)
```

Comment:
But if we use the reused project.  The donwloaded might be larger than the uploaded.